test/src/single_thread_matrix_calculation_test.cpp

```diff
@@ -161,14 +161,44 @@ TEST_F(TestSinglThreadCalculation, subtractNumberSubMatrix) {
     Matrix<> result2({{-1, -1, -1}, {7, 6, 5}, {3, 3, 3}});
     ASSERT_TRUE(equalSingleThread(output2, result2, 0, 0, {3, 3}));
 }
-// TODO
-TEST_F(TestSinglThreadCalculation, multiplyNumberWholeMatrix) {}
-// TODO
-TEST_F(TestSinglThreadCalculation, multiplyNumberSubMatrix) {}
-// TODO
-TEST_F(TestSinglThreadCalculation, divideNumberWholeMatrix) {}
-// TODO
-TEST_F(TestSinglThreadCalculation, divideNumberSubMatrix) {}
+
+TEST_F(TestSinglThreadCalculation, multiplyNumberWholeMatrix) {
```

Comment:
See the test name, multiplyNumber, you should test this function:
![image](https://github.com/Kaiser-Yang/matrix-calculation-accelarator/assets/58209855/734d5305-b6f7-4eb8-82e2-6ce7298f7307)